main.tf

```diff
@@ -150,10 +164,58 @@ resource "kubernetes_deployment" "wandb" {
             }
           }
         }
+
+        container {
+          name = "sidecar"
+          image = "busybox"
+
+          volume_mount {
+            name = "gorilla"
+            mount_path = "/var/log/${local.gorilla}"
+            sub_path = local.gorilla
+          }
+
+          volume_mount {
+            name = "local" 
+            mount_path = "/var/log/${local.local}"
+            sub_path = local.local
+          }
+
+          args = [ "/bin/sh", "-c", "tail -n+1 -f /var/log/gorilla.log" ]
+        }
+
         volume {
           name = local.app_name
           config_map {
             name     = kubernetes_config_map.config_map.metadata[0].name
+            items {
+              key = "server_ca.pem"
+              path = "server_ca.pem"
+            }
```

Comment:
Curious why this was added?  If we need it we should probably use `local.redis_ca_cert_name` here instead of hard coding the name of the pem.

Reply:
whoops was messing around with some of the configurations but reverting this later